### PR TITLE
remove falcon iframe routes

### PR DESCRIPTION
**Pull request by callumlocke**
Tidyup to get rid of Falcon-only stuff.

(Also bumps dependencies and fixes new lint issues.)

### CODE REVIEW

**Comment by kavanagh** [comment]
LGTM!!

File: src/server/server.js [hunk 1]
```diff
@@ -144,70 +139,6 @@ router
...
-			return;
-		}
-	})
-
-	// iframe (for using on the Falcon brexit page)
-	.get('/brexit/iframe.html', async ctx => {
-		ctx.set('Cache-Control', 'max-age=500');
-
-		try {
-			ctx.body = renderBrexitIframe(await getBrexitLocals());
```
Comment (kavanagh) [inline src/server/server.js:168]: Sending a 410 Gone is perhaps more correct.

File: src/server/server.js [hunk 2]
```diff
@@ -144,70 +139,6 @@ router
...
-			return;
-		}
-	})
-
-	// iframe (for using on the Falcon US Election page)
-	.get('/us-election-2016/iframe.html', async ctx => {
-		ctx.set('Cache-Control', 'max-age=500');
-
-		try {
-			ctx.body = renderUsElectionIframe(await getUSElectionLocals());
```
Comment (kavanagh) [inline src/server/server.js:200]: Ditto 410 code

File: src/server/server.js (hunk 1 above)
Comment (callumlocke) [reply src/server/server.js:168]: what would this affect?

**Pull request merged**